devops/deploy/spin-up-script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
```

Comment:
Consider taking the number of ports as an input to the script, but have a default.

Reply:
Done!